Pipfile

```diff
@@ -20,13 +20,15 @@ markdown-headdown = "==0.1.3"
 psycopg2-binary = "==2.9.9"
 sentry-sdk = "==1.45.0"
 sorl-thumbnail = "==12.10.0"
-Django = "==4.2.11"
 Pillow = "==10.3.0"
 Markdown = "==3.3.7"
 feedparser = "==6.0.11"
 pytz = "==2024.1"
 urllib3 = "<3.0.0"
 pre-commit = "*"
+django = "==4.2.15"
+zipp = "*"
+certifi = "*"
```

Comment:
Wow. Pipenv makes it really hard to upgrade only one package in the lockfile these days

- `ValueError: The flag --selective-upgrade flag has been removed`
- `ValueError: The flag --keep-outdated flag has been removed`
- `--lock-only` exists, but the doesn't do what it says it will